src/networking.c

```diff
@@ -521,6 +521,15 @@ void afterErrorReply(client *c, const char *s, size_t len, int flags) {
             showLatestBacklog();
         }
         server.stat_unexpected_error_replies++;
+
+        /* If this command was from our master, we are not a writable replica,
+         * check to see if we should panic based off the replication error
+         * behavior config. */
+        if (ctype == CLIENT_TYPE_MASTER && server.repl_slave_ro &&
+            server.repl_error_behavior == REPLICATION_ERR_BEHAVIOR_PANIC)
+        {
+            serverPanic("Error detected on replication stream.");
```

Comment:
i think it will be much more convenient if all the details (`from`, `to`, `cmdname`) appeared in the crash log itself rather than further up in the log